var/spack/repos/builtin/packages/hpe-mpi/package.py

```diff
@@ -42,13 +47,9 @@ def install(self, spec, prefix):
         for mpic in find(self.stage.source_path, 'mpic*'):
             mode = os.stat(mpic).st_mode
             os.chmod(mpic, mode | stat.S_IWRITE)
-            filter_file(r'-I(.*mpiroot)', r'-isystem\1', mpic)
```

Comment:
Didn't see this before, but this should stay in. Bad things happen with these stupid compiler wrappers and over-zealous C developers.

Reply:
As a workaround, we can change that to `-${DUMB_COMPILER:+I}${DUMB_COMPILER:-isystem}`. Then define `DUMB_COMPILER` in Spack's dependent build env for `nvhpc` and all others that need it. This is also the nicest naming of these variables that I will agree to.

@ohm314 as original author, thoughts?